gadopt/diagnostics.py

```diff
@@ -41,10 +50,10 @@ class GeodynamicalDiagnostics:
 
     def __init__(
         self,
-        z: Function,
-        T: Function,
         bottom_id: int,
         top_id: int,
+        z: Function,
+        T: Function = 0,
         quad_degree: int = 4,
     ):
```

Comment:
Here, the call signature should start with `z, T=0, bottom_id=None, top_id=None`, with appropriate changes in `__init__`.